Form-Controls/index.html

```diff
@@ -13,15 +13,61 @@
     <h1>Product Pick</h1>
   </header>
   <main>
+
   <form>
+
     <!-- write your html here-->
+    
+      <label for="firstname">First name:</label>
+      <input type="text" name="firstname" id="firstname" placeholder="Enter your name" required>
+<br>
+      <label for="lastname">Last name:</label>
+      <input type="text" name="lastname" id="lastname" placeholder="Enter your lastname" required>
+<br>
+      <label for="Email">Email:</label>
+      <input type="email" name="Email" id="Email" required placeholder="Enter your Email">
+<br>
```

Comment:
```suggestion

```

What are these br tags doing? Are they doing layout? Let's try to organise the layout with CSS and describe the data clearly with semantic html.